controllers/provisioners/eks/helpers.go

```diff
@@ -97,8 +97,19 @@ func (ctx *EksInstanceGroupContext) ResolveSecurityGroups() []string {
 }
 
 func (ctx *EksInstanceGroupContext) GetBasicUserData(clusterName, args string, payload UserDataPayload, mounts []MountOpts) string {
-
-	var UserDataTemplate = `#!/bin/bash
+	osFamily := ctx.GetInstanceGroup().Labels[OsFamilyLabel]
```

Comment:
This should work right?
```suggestion
      osFamily := ctx.GetOsFamily()
```